sensor_msgs/Image image
---
face_classification/ClassificationResult[] results
